tapalcatl_server/handler.go

```diff
@@ -116,6 +121,10 @@ func logBool(x bool) string {
 	}
 }
 
+func logDuration(x time.Duration) string {
+	return fmt.Sprintf("%d", x.Nanoseconds())
```

Comment:
I used nanoseconds to be more precise because it'll just be 0 for some metrics.

We can:

* accept having 0s
* get rid of the metrics that are only producing 0s
* use milliseconds but a floating point value
* keep nanoseconds